### DISCUSSION TITLE
How do I specify for a figure/table to be unnumbered?

### DISCUSSION THREAD

**Discussion by meryltheng** [quarto-dev/quarto-cli#4287] [Q&A]
Is this possible? I understand I can do it with Section headers with {.unnumbered}. It does not seem to work with figures and tables.

**Comment by rgaiacs** [comment]
Because of [Cross References](https://quarto.org/docs/authoring/cross-references.html), figures and tables must be numbered. This is similar to Pandoc's default behaviour:

The minimal working example

```
$ pandoc --from markdown --to html <<EOF
![Foo](bar.jpg)
EOF
```

produces

```
<figure>
<img src="bar.jpg" alt="Foo" />
<figcaption aria-hidden="true">Foo</figcaption>
</figure>
```

If you want to have unnumbered figures, you should try "[regular inline image](https://pandoc.org/MANUAL.html#images)":

The minimal working example, **that includes a nonbreaking space** in the form of `\ `,

```
$ pandoc --from markdown --to html <<EOF
![Foo](bar.jpg)\ 
EOF
```

produces

```
<p><img src="bar.jpg" alt="Foo" /> </p>
```

Please give a try.
